tests/features/support/steps.js

```diff
@@ -146,14 +146,18 @@ Then('I should see the link {string}', async (linkText) => {
   expect(link).to.exist;
 });
 
-// TODO: Research how da handles invalid symbols
-Then('the question id should be {string}', async (question_class) => {
-  /* Looks for the question id as it's written in the .yml.
+// TODO: Switch to getting the id and then checking it against the argument
+//     which will make for more useful test error messages.
+Then('the question id should be {string}', async (question_id) => {
+  /* Looks for a santized version of the question id as it's written
+  *     in the .yml.
+  *
+  * docassemble: re.sub(r'[^A-Za-z0-9]+', '-', interview_status.question.id.lower())
   *  
   *  WARNING: Does not handle actual html class attribute name on page
   */
-  question_class = question_class.replace(/\s/g, '-');
-  question_class = 'question-' + question_class;
+  clean_id = question_id.toLowerCase().replace(/[^A-z0-9]+/g, '-');
```

Comment:
I think the regex might be more robust if it's written as `A-Za-z` instead of `A-z`. As I understand it, this relies on ASCII codes (?). `A-z` in ASCII will cover non-alphabetical characters like [, ], and \.